tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml

```diff
@@ -347,6 +335,22 @@ spec:
         - '{policy: .policy, key: .key, "effective-time": .["effective-time"]}'
         - "$(params.HOMEDIR)/report-json.json"
 
+    - name: detailed-report
+      image: quay.io/conforma/cli:latest
+      command: ["sh", "-c"]
+      args:
+        - |
+          # Check if the report file exists before trying to display it
+          if [ -f "$(params.HOMEDIR)/detailed-text-report.txt" ]; then
+            cat "$(params.HOMEDIR)/detailed-text-report.txt" || true
+          else
+            echo "Warning: detailed-text-report.txt not found. Ensure validate step completed successfully and produced the file."
+          fi
+      volumeMounts:
+        - mountPath: /var/workdir
+          name: workdir
```

Comment:
I think this is what's causing the acceptance test failure. There's no such volume defined in this task.